src/Helpers/Click/ClickedEditArtwork.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  ActionType,
+  ClickedEditArtwork,
+  ContextModule,
+  PageOwnerType,
+} from "../../Schema"
+
+export interface ClickedEditArtworkArgs {
+  contextPageOwnerType: PageOwnerType
+  contextModule: ContextModule
+  destinationPageOwnerType: PageOwnerType
+  destinationPageOwnerId: string
+  destinationPageOwnerSlug: string
+  destinationPath: string
+}
+
+/**
+ * A user clicks to edit an artwork on web.
+ *
+ *  @example
+ *  ```
+ *  clickedExpansionToggle({
+ *    contextModule: ContextModule.toDoList,
+ *    contextPageOwnerType: OwnerType.home,
+ *    destinationPageOwnerType: OwnerType.artwork,
+ *    destinationPageOwnerId: "5ff5c57b5022f600116d7a1a",
+ *    destinationPageOwnerSlug: "daniel-heidkamp-el-capitan-night",
+ *    destinationPath: "/artworks/daniel-heidkamp-el-capitan-night/edit"
```

Comment:
And one other question for you @brendamanganelli - I noticed that `destination_path` is in the interface in the schema but not the example, nor is it in the spreadsheet. Do we want it?

Reply:
Yes, it would be great to have it, I apologise for making it unclear - I'll amend the spreadsheet. Thanks for flagging this!

Reply:
No worries, thanks for clarifying!